src/web/mjs/connectors/Novelgo.mjs

```diff
@@ -0,0 +1,93 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class Novelgo extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'novelgo';
+        super.label = 'Novelgo.ID';
+        this.tags = [ 'novel', 'indonesian' ];
+        this.url = 'https://novelgo.id';
+        this.path = '/genre/page/';
+
+        this.queryMangas = 'div.novel-item-content h3.novel-item-title a';
+        this.novelContentQuery = 'div.noveils-chapter-wrapper div#chapter-post-content';
+        this.novelObstaclesQuery = 'div.code-block';
+        this.novelFormat = 'image/png';
+        this.novelWidth = '56em'; // parseInt(1200 / window.devicePixelRatio) + 'px';
+        this.novelPadding = '1.5em';
+    }
+
+    async _getMangasFromPage(page) {
+        const URI = new URL(this.path + page + '/', this.url);
+        let request = new Request(URI, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryMangas);
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim()
+            };
+        });
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        for(let page = 1, run = true; run; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangas.length > 0 ? mangaList.push(...mangas) : run = false;
+        }
+        return mangaList;
+    }
+
+    async _getChaptersFromJSON(page, id){
+        const URI = new URL('/wp-json/noveils/v1/chapters?paged=' + page + '&perpage=250&category=' + id, this.url);
```

Comment:
May set URL paramaters explicitly for better readibility
Use lower case names for variables, `URI` and `URL` can be confusing when reading the code
```javascript
const uri = new URL('/wp-json/noveils/v1/chapters', this.url);
uri.searchParams.set('paged', page);
// ... more param setters
```